### ISSUE TITLE
Alternative redundant data sources to back up OpenStates, Google Civic?

### ISSUE THREAD

**Issue by dannewoo** [OpenSourceActivismTech/call-power#96]
It looks like the Open States API has been down since August 25th. I propose adding in Google Civic API as an alternative to the zip code look up for local representatives. This API also doesn’t require 3rd party zip code geocoding but is included with the API.

**Comment by jlev** [comment]
Google Civic might be a good replacement, but at least in my test queries it does not include representative info for state legislatures, which is what we are currently using OpenStates for. Strangely, it does include local positions like Mayor and County DA, including phone contacts.

Check out the spec at https://developers.google.com/civic-information/docs/v2/representatives/representativeInfoByAddress and let me know if you see a way forward with it.

**Comment by dannewoo** [comment]
Thanks @jlev thats great news! Indeed I too have sympathy for their team, lets hope some great organization decides to through gobs of money at this project soon to help.

On further review of the Google Civic API I realized that they only have some local level representatives listed in their database, but its a bit spotty so probably (at least not at this point) won’t be a good alternative to Open States. I made a simple web app that uses both of these API’s so we can check to see what Google Civic offers versus Open States offers on the local level based on a address. Feel free to check it out: https://remix4change.glitch.me/

Thanks again! Call Power is amazing!
